### PR TITLE
modified error 404 only 

### PR DESCRIPTION
**Pull request by abhishekpanthee**
@arjunatapadkar  yup pushed only error page
UI/UX Redesign for Improved cheats Interface for dark mode as well as fix for non-functioning footer  
i am really new to it so 
should i pull new pr  after you merge this (if good enough )as you said step by step 
your guidance would be helpful

### CODE REVIEW

**Review by arjunatapadkar — APPROVED** [review]

**Comment by abhishekpanthee** [comment]
@arjunatapadkar   so my issue was UI/UX Redesign for Improved cheats Interface for dark mode as well as fix for non-functioning footer
i am really new to it so
should i pull new pr for each feature  after you merge this (if good enough )as you said step by step
your guidance would be helpful

**Comment by arjunatapadkar** [comment]
@abhishekpanthee  You can make multiple PRs for different issues, it will help you as well to increase your scores
And add snapshots of improvements in UI along with PRs

**Pull request merged**
